CHANGES

```diff
@@ -68,6 +68,8 @@ Universes
   module and library boundaries. Global universe names introduced in an
   inductive / constant / Let declaration get qualified with the name of
   the declaration.
+- Notations that start with `Type` now support universe instances with
+  `@{u}` (fixing GH-5726).
```

Comment:
Please use the standard "#5726" instead. Thanks!

Reply:
Done - I actually wrote BZ#5726 since this does originally come from Bugzilla.